report/firefox_code_coverage/codecoverage.py

```diff
@@ -310,6 +313,27 @@ def download_grcov():
     return local_path
 
 
+def upload_html_report(
+    report_dir, base_artifact="public/report", ttl=timedelta(days=10)
+):
+    assert os.path.isdir(report_dir), "Not a directory {}".format(report_dir)
+    report_dir = os.path.realpath(report_dir)
+    assert not base_artifact.endswith("/"), "No trailing / in base_artifact"
+
+    # Use Taskcluster proxy when available
+    taskcluster.auth()
+
+    for path in Path(report_dir).rglob("*"):
+
+        filename = str(path.relative_to(report_dir))
+        content_type = magic.from_file(str(path), mime=True)
+        print("Uploading {} as {}".format(filename, content_type))
```

Comment:
logger.debug, or this will print a lot of stuff

Reply:
Done